rhasspyhermes_app/__init__.py

```diff
@@ -106,6 +107,11 @@ def __init__(
         # Parse command-line arguments
         self.args = parser.parse_args()
 
+        # Option to set all parameters as keyword arguments !!! CLI arguments are rated higher priority
+        args_dict = vars(self.args)
+        for key, value in kwargs.items():
+            args_dict[key] = args_dict[key] if key in args_dict else value
```

Comment:
I think this should be simply written as:
```python
args_dict = {**kwargs, **vars(self.args)}
```
instead

Reply:
This is so much more cleaner in every way jeeeeez.... i didn't even know of this feature...
```suggestion
        # Option to set all parameters as keyword arguments !!! CLI arguments are rated higher priority
        args_dict = {**kwargs, **vars(self.args)}
```